spec/unit/default_spec.rb

```diff
@@ -1,240 +1,23 @@
-
-            expect(chef_run).to render_file('/etc/grafana/ldap.toml').with_content(/^\[\[servers\]\]/)
-            expect(chef_run).to render_file('/etc/grafana/ldap.toml').with_content(/^host = "127.0.0.1"/)
-            expect(chef_run).to render_file('/etc/grafana/ldap.toml').with_content(/^port = 389/)
-            expect(chef_run).to render_file('/etc/grafana/ldap.toml').with_content(/^use_ssl = false/)
-            expect(chef_run).to render_file('/etc/grafana/ldap.toml').with_content(/^ssl_skip_verify = false/)
-            expect(chef_run).to render_file('/etc/grafana/ldap.toml').with_content(/^bind_dn = "cn=admin,dc=grafana,dc=org"/)
-            expect(chef_run).to render_file('/etc/grafana/ldap.toml').with_content(/^bind_password = grafana/)
-            expect(chef_run).to render_file('/etc/grafana/ldap.toml').with_content(/^search_filter = "\(cn=%s\)"/)
-            expect(chef_run).to render_file('/etc/grafana/ldap.toml').with_content(/^search_base_dns = \["dc=grafana,dc=org"\]/)
-
-            expect(chef_run).to render_file('/etc/grafana/ldap.toml').with_content(/^\[servers.attributes\]/)
-            expect(chef_run).to render_file('/etc/grafana/ldap.toml').with_content(/^name = "givenName"/)
-            expect(chef_run).to render_file('/etc/grafana/ldap.toml').with_content(/^surname = "sn"/)
-            expect(chef_run).to render_file('/etc/grafana/ldap.toml').with_content(/^username = "cn"/)
-            expect(chef_run).to render_file('/etc/grafana/ldap.toml').with_content(/^member_of = "memberOf"/)
-            expect(chef_run).to render_file('/etc/grafana/ldap.toml').with_content(/^email = "email"/)
-
-            expect(chef_run).to render_file('/etc/grafana/ldap.toml').with_content(/^\[\[servers.group_mappings\]\]/)
-            expect(chef_run).to render_file('/etc/grafana/ldap.toml').with_content(/^group_dn = "cn=admins,dc=grafana,dc=org"/)
-            expect(chef_run).to render_file('/etc/grafana/ldap.toml').with_content(/^org_role = "Admin"/)
-            expect(chef_run).to render_file('/etc/grafana/ldap.toml').with_content(/^# org_id = 1/)
-
-            expect(chef_run).to render_file('/etc/grafana/ldap.toml').with_content(/^\[\[servers.group_mappings\]\]/)
-            expect(chef_run).to render_file('/etc/grafana/ldap.toml').with_content(/^group_dn = "cn=users,dc=grafana,dc=org"/)
-            expect(chef_run).to render_file('/etc/grafana/ldap.toml').with_content(/^org_role = "Editor"/)
-
-            expect(chef_run).to render_file('/etc/grafana/ldap.toml').with_content(/^\[\[servers.group_mappings\]\]/)
-            expect(chef_run).to render_file('/etc/grafana/ldap.toml').with_content(/^group_dn = "*"/)
-            expect(chef_run).to render_file('/etc/grafana/ldap.toml').with_content(/^org_role = "Viewer"/)
-          end
-
-          it 'enable grafana-server service' do
-            expect(chef_run).to enable_service('grafana-server')
-          end
-        end
+platforms = {
```

Comment:
For a better time testing custom resources look at https://github.com/sous-chefs/apache2/tree/master/spec/resources